routes/virtual.js

```diff
@@ -38,12 +38,18 @@ function registerRRPair(service, rrpair) {
     }
     else {
       debug("HTTP methods don't match");
-      return next();
+      if(matched)
```

Comment:
In this case, the variable `match` is always false. This is because it is initialized as false on line 22, and is only set to true if the request methods match (lines 28-38). 

Therefore, this entire if / else block (lines 41-44) should be reduced to:

```
return next(new Error("HTTP methods does not match")); 
```